activestate.yaml

```diff
@@ -48,6 +48,7 @@ constants:
 scripts:
   - name: install-deps-dev
     language: bash
+    standalone: true
```

Comment:
Because I find often while testing that it re-sources my runtime because I invoked it with another state tool version (ie. the one I'm working on vs my installed state tool). And end of the day none of our scripts actually require the runtime, they were always standalone, just never marked as such.